ptp/ptp4u/server/subscription.go

```diff
@@ -104,14 +104,14 @@ func (sc *SubscriptionClient) Start(ctx context.Context) {
 	sc.runningInterval = sc.interval
 	sc.intervalTicker = time.NewTicker(sc.runningInterval)
 
-	defer sc.intervalTicker.Stop()
 	defer sc.setRunning(false)
+	defer sc.sendSignalingCancel()
+	defer sc.intervalTicker.Stop()
 
 	for {
 		select {
 		case <-ctx.Done():
 			log.Infof(over)
-			// TODO send cancellation
```

Comment:
There is another comment like this in line 133. Should it be looked at too?

Reply:
Yeah it was just a reminder. It would go via flow above.